mk/help.mk

```diff
@@ -30,6 +30,7 @@ help:
 	@echo "  vrbrain - the VRBrain boards"
 	@echo "  sitl - the SITL Software In The Loop simulation"
 	@echo "  bbbmini - the Beagle Bone Black mini"
+	@echo "  bbblue - the BeagleBone Blue"
```

Comment:
@jadonk I have to prepare a copter with the Blue and wait for appropriate weather for a test flight. It is tested on my desk until now.